sklearn/feature_selection/_univariate_selection.py

```diff
@@ -238,8 +238,8 @@ def f_regression(X, y, center=True):
     This is done in 2 steps:
 
     1. The correlation between each regressor and the target is computed,
-       that is, ((X[:, i] - mean(X[:, i])) * (y - mean_y)) / (std(X[:, i]) *
-       std(y)).
+       that is, mean((X[:, i] - mean(X[:, i])) * (y - mean(y))) / (std(X[:, i])*       std(y)).
+
```

Comment:
To pass the linter we need to be < 80 characters per line:

```
    1. The correlation between each regressor and the target is computed,
       that is, mean((X[:, i] - mean(X[:, i])) * (y - mean(y))) / (std(X[:, i])
       * std(y)).
```

Reply:
Resolved in PR #22252